research/gan/mnist/util.py

```diff
@@ -24,7 +24,6 @@
 
 
 import numpy as np
-from six.moves import xrange
```

Comment:
Hey Joel, we should keep this line in order to maintain Python 3 compatibility. Same for the other files.